modules/sievefilters/modules.php

```diff
@@ -1344,46 +1346,3 @@ public function process() {
         }
     }
 }
-
-/**
- * @subpackage sievefilterstoggle/handler
- */
-class Hm_Handler_sieve_toggle_script_state extends Hm_Handler_Module {
```

Comment:
You are removing this but i can't see the corresponding line removed in setup.php